lib/apiForSheets.js

```diff
@@ -18,14 +18,15 @@ var clientSecret = process.env.CLIENT_SECRET;
 var context;
 
 /**
-  * Create an OAuth2 client with the given credentials, and then execute the
-  * given callback function.
-  *
-  * @param {Object} credentials The authorization client credentials.
-  * @param {function} callback The callback to call with the
-  *   authorized client.
-  */
-module.exports.authorize = function(credentials) {
+ * Create an OAuth2 client with the given credentials, and then execute the
+ * given callback function.
+ *
+ * @param {Object} credentials The authorization client credentials.
+ * @param {function} callback The callback to call with the
+ *   authorized client.
+ */
+/* istanbul ignore next */
```

Comment:
Just to confirm - this comment ignores the authorize function for coverage check, right?

Also, on further thoughts, why can't we pass a mock credentials object and mock the OAuth2 api calls?